frontend/plan/src/components/search/SearchBar.js

```diff
@@ -37,6 +37,90 @@ function shouldSearch(filterData) {
     return searchString || selectedReq;
 }
 
+const LoginButton = () => (
+    <a
+        className="button is-link login"
+        href={"/accounts/login/?next=" + window.location}
+        style={{
+            padding: "0.5rem",
+            fontSize: "1rem!important",
+            paddingRight: "1rem",
+            paddingLeft: "1rem",
+        }}
+    >
+        Login
+    </a>
+);
+
+const UserSelector = ({ user: { first_name, last_name, username } }) => {
```

Comment:
The "click outside" logic of this component can be abstracted away with useOnClickOutside.js